shared/actions/chat2/index.js

```diff
@@ -845,6 +845,16 @@ const loadMoreMessages = (
       ''} num: ${numberOfMessagesToLoad} reason: ${reason}`
   )
 
+  const rpcReason = (function(r) {
```

Comment:
Just use let and not do an inline function call

Reply:
How come? This seemed shorter.

Reply:
Then hoist it up and call it

Reply:
Is it just a style thing, or is this dangerous? I'll move it, just curious.

Reply:
It's a style thing. We never use iife.